backend/src/apiserver/common/config.go

```diff
@@ -23,14 +23,15 @@ import (
 )
 
 const (
-	MultiUserMode                       string = "MULTIUSER"
-	MultiUserModeSharedReadAccess       string = "MULTIUSER_SHARED_READ"
-	PodNamespace                        string = "POD_NAMESPACE"
-	CacheEnabled                        string = "CacheEnabled"
-	KubeflowUserIDHeader                string = "KUBEFLOW_USERID_HEADER"
-	KubeflowUserIDPrefix                string = "KUBEFLOW_USERID_PREFIX"
-	UpdatePipelineVersionByDefault      string = "AUTO_UPDATE_PIPELINE_DEFAULT_VERSION"
-	TokenReviewAudience                 string = "TOKEN_REVIEW_AUDIENCE"
+	MultiUserMode                           string = "MULTIUSER"
+	MultiUserModeSharedReadAccess           string = "MULTIUSER_SHARED_READ"
+	PodNamespace                            string = "POD_NAMESPACE"
+	CacheEnabled                            string = "CacheEnabled"
+	DefaultPipelineRunnerServiceAccountFlag string = "DEFAULTPIPELINERUNNERSERVICEACCOUNT"
```

Comment:
nit: For the naming of both the constant variable and the key/flag used in config.json, do you think we need to carry `Default` as a prefix? 
How about just `PipelineRunnerServiceAccountFlag string = "PIPELINE_RUNNER_SERVICE_ACCOUNT"` or `PipelineRunnerServiceAccountKey string = "PIPELINE_RUNNER_SERVICE_ACCOUNT"`?

Reply:
To explain why it had the default prefix: because users can also specify service account when creating runs. The env var is the default.

+1 for changing the name to make it more readable. Ideally, when we change the name, also read from the legacy env var as fallback.

Reply:
I think we need to set the default prefix because user can override service account when creating Pipeline Run. https://screenshot.googleplex.com/AApqJMAg7ikYpnZ.

More importantly, changing the env var name is going to be a breaking change as Kubeflow manifests and All distributions rely on the existing name. (Note: We are not creating a new variable, I am only bringing back the old variable that was removed accidentally in this file https://github.com/kubeflow/pipelines/pull/6689/files#diff-47d6ab6bca869e0559465170316707ceda42f65103bd0d4e93519294305e62d1)

See: https://github.com/kubeflow/pipelines/blob/74c7773ca40decfd0d4ed40dc93a6af591bbc190/manifests/kustomize/base/installs/multi-user/api-service/params.env

Reply:
I think all configs are meant for default and overridable, thus the prefix "Default" is redundant.
But I see your point about avoiding breaking changes. SGTM.